runner/src/server/plugins/engine/pageControllers/RepeatingSummaryPageController.ts

```diff
@@ -110,14 +110,22 @@ export class RepeatingSummaryPageController extends PageController {
     };
   }
 
-  buildRows(state) {
+  buildRows(state, response) {
+    let form_session_identifier = "";
+    if (response.request.query.form_session_identifier) {
+      form_session_identifier = `&form_session_identifier=${response.request.query.form_session_identifier}`;
+    }
+
     if (this.inputComponent.type === "MultiInputField") {
-      return this.buildTextFieldRows(state);
+      return this.buildTextFieldRows(state, form_session_identifier);
     }
-    return this.getRowsFromAnswers(state);
+    return this.getRowsFromAnswers(
+      state,
+      response.query.form_session_identifier
```

Comment:
Is this meant to be `response.query.form_session_identifier`? it differs from the `response.request.query.form_session_identifier` above, what is different between them two?